src/train.py

```
@@ -0,0 +1,184 @@
+
+    RUN_RESULTS_DIR = os.path.join(BASE_RESULTS_DIR, RUN_NAME)
+    os.makedirs(RUN_RESULTS_DIR, exist_ok=True)
+
+    print(f"\n--- ÚJ FUTTATÁS INDUL ---")
+    print(f"Kísérlet neve: {RUN_NAME}")
+    print(f"Minden eredmény ide lesz mentve: {RUN_RESULTS_DIR}")
+
+    # --- 2. Adatok betöltése ---
+    print("Adatok betöltése...")
+    data = load_data()
+    if data is None: return
```

Ruff (0.14.5)

Multiple statements on one line (colon)

(E701)